.gitattributes

```diff
@@ -1 +1,27 @@
-* text=auto eol=lf
\ No newline at end of file
+## GITATTRIBUTES FOR WEB PROJECTS
+#
+# These settings are for any web project.
+#
+# Details per file setting:
+#   text    These files should be normalized (i.e. convert CRLF to LF).
+#   binary  These files are binary and should be left untouched.
+#
+# Note that binary is a macro for -text -diff.
+######################################################################
+
+# Auto detect
+##   Handle line endings automatically for files detected as
+##   text and leave all files detected as binary untouched.
+##   This will handle all files NOT defined below.
+*                 text=auto
```

Comment:
I think we're missing the `eol=lf` option

Reply:
text=auto does this

Reply:
True 😅